### PR TITLE
Distr h liu

### PR DESCRIPTION
**Pull request by hliu26**

### CODE REVIEW

**Comment by kevindavisross** [comment]
I haven't yet done a thorough review, but what I have tried so far seems to be working fine.

File: symbulate/results.py [hunk 1]
```diff
@@ -441,6 +442,104 @@ def standardize(self):
...
+            return tuple(stsmodel.mad(a=np.array(self), c=1, axis=0))   
+        else:                                                                        
+            raise Exception("I don't know how to take the median absolute deviation of these values.")
+
+    def iqr(self):
+        if all(is_scalar(x) for x in self):                                          
+            q75, q25 = np.percentile(np.array(self), [75, 25])
+            return q75-q25
+        elif get_dimension(self) > 0:                                                
+            return tuple(np.subtract(np.percentile(np.array(self), 75, axis = 0),
```
Comment (dlsun) [inline symbulate/results.py]: No space around = sign.

File: symbulate/results.py [hunk 2]
```diff
@@ -441,6 +442,104 @@ def standardize(self):
...
+        if all(is_scalar(x) for x in self):
+            return np.median(np.array(self))
+        elif get_dimension(self) > 0:
+            return tuple(np.median(np.array(self), 0))
+        else:
+            raise Exception("I don't know how to take the median of these values.")
+
+    def quantile(self, q):
+        if all(is_scalar(x) for x in self):
+            return np.percentile(np.array(self), q*100)
```
Comment (dlsun) [inline symbulate/results.py]: Please add a space around mathematical operations like +, -, *, /. So this should be `q * 100`, not `q*100`.

**Comment by dlsun** [comment]
Howard, this looks fine to me, but could you please upload a notebook with all of your tests verifying that all of these functions work?

**Comment by hliu26** [comment]
Here are examples of the functions working for .apply() and being called with RV .function()
[UnitTest_Math2.pdf](https://github.com/dlsun/symbulate/files/1290105/UnitTest_Math2.pdf)

**Comment by dlsun** [comment]
Thanks! Could you please fix the style errors that I noted above and then merge?

**Comment by kevindavisross** [comment]
Is this branch ready to be merged?  If so, there are some conflicts that need to be resolved before merging since I have now merged in Robert's branch.  It looks like just a few lines that are causing the problem, but I'm afraid I'll break it if I try to fix it.

**Comment by kevindavisross** [comment]
What's the status of this pull request?  I think it is ready to be merged in, but there are some conflicts that need to be resolved.  Let me know if you make any changes and I can rerun my checks.

**Comment by kevindavisross** [comment]
@dlsun : Can we merge this pull request into master?  This is Howard's pull request that implemented a lot of summary statistic functions (median, quantile, moment, etc) in both RV space and in simulation space.  At the time I think there were conflicts between this and what Robert was working on, so this wasn't merged.  But there no longer seem to be any conflicts between this pull request and Symbulate master.  

It looks like you had some style issues with the code, which Howard mentions he fixed.  If the code is acceptable to you, can you merge in?

I just reran my test of these functions and everything was fine; pdf of the notebook is attached.  I also ran the unit tests and they checked out.
[Test - summary statistics in RV and simulation space.pdf](https://github.com/dlsun/symbulate/files/2275540/Test.-.summary.statistics.in.RV.and.simulation.space.pdf)

**Pull request merged**